packages/scss/src/components/_table.scss

```diff
@@ -110,22 +109,27 @@
 
 .table-body-row-cell, .table-head-row-cell {
 	&.mod-actions {
-		opacity: 0;
 		padding: {
 			bottom: .3rem;
 			top: .3rem;
 		}
 		text-align: right;
 		transition: opacity _theme("commons.animations.durations.fast") ease;
 		white-space: nowrap;
+
+		* {
```

Comment:
I guess this is to have better control on what is invisible ? If so, shouldn't you use a ` > *` or a specific class to only affect direct children ? It feels like it could change opacity for a lot for things this way

Reply:
Without border collapse TD's border was also invisible.
I can also use a direct selector. But as we are inside a very specific class, mod-action (which format line actions), using another class seems to be a bit painful to use.